docs/app-gloss.md

```diff
@@ -2,107 +2,107 @@
+The provenance or type of Event Data. Data sources include Wikipedia, Crossref, DataCite etc. A Source is different to an Agent, which is a piece of software that fetches data for a particular data source.
 
 ### Event UUID
 
- A UUID that corresponds to an Event within Crossref Event Data. 
+A UUID that corresponds to an Event within Crossref Event Data. 
 
 ### Registration Agency
 
- An organization that assigns DOIs. For example, Crossref and DataCite.
+An organization that assigns DOIs. For example, Crossref and DataCite.
 
-### Registered Content
+### Registered content
 
- Content that has been registered with Crossref and assigned a Crossref DOI, e.g an article or book chapter.
+Content that has been registered with Crossref and assigned a Crossref DOI, e.g an article or book chapter.
 
 ### UUID
 
- Universally Unique Identifier. Essentially a random number that identifies an Event. Looks like `c0eb1c46-6a59-49c9-926b-a10667ddd9de`.
+Universally Unique Identifier. Essentially a random number that identifies an Event. Looks like `c0eb1c46-6a59-49c9-926b-a10667ddd9de`.
 
 # Appendix: Abbreviations
 
 ### API
 
- Application Programming Interface. An interface for allowing one piece of software to connect to another. Event Data collects data from other APIs from other services and provides an API for allowing access to data.
+Application Programming Interface. An interface for allowing one piece of software to connect to another. Event Data collects data from other APIs from other services and provides an API for allowing access to data.
 
 ### Event Data
 
- Crossref Event Data
+Crossref Event Data
 
 ### CoC
 
- Code of Conduct
+Code of Conduct
 
 ### DET
 
- Means 'DOI Event Tracking', the original name for Crossref Event Data.
+Means 'DOI Event Tracking', the original name for Crossref Event Data.
 
 ### DOI
 
- Digital Object Identifier. An identifier given to a Registered Content Item, e.g. http://doi.org/10.5555/12345678
+Digital Object Identifier. An identifier given to a Registered Content Item, e.g. http://doi.org/10.5555/12345678
 
 ### JSON 
 
- JavaScript Object Notation. A common format for sending data. All data coming out of the Event Data API is in JSON format.
+JavaScript Object Notation. A common format for sending data. All data coming out of the Event Data API is in JSON format.
 
 ### MEDRA
 
- Multilingual European DOI Registration Agency. A DOI Registration Agency.
+Multilingual European DOI Registration Agency. A DOI Registration Agency.
 
 ### NISO
 
- National Information Standards Organization. A standards body who have created a Code of Conduct for altmetrics.
+National Information Standards Organization. A standards body who have created a Code of Conduct for altmetrics.
 
 ### ORCiD
 
- Open Researcher and Contributor ID. A system for assigning identifiers to authors.
+Open researcher and contributor ID. A system for assigning identifiers to authors.
```

Comment:
Shouln't these be capitals to explain the acroynm? In any case it looks like ORCID is trying to get away from that: https://orcid.org/trademark-and-id-display-guidelines